lib/src/Uncertainty/StatTests/VisualTest.cxx

```diff
@@ -297,6 +297,36 @@ GridLayout VisualTest::DrawPairs(const Sample & sample)
   return grid;
 }
 
+/* Draw 2-d projections of a multivariate sample */
+GridLayout VisualTest::DrawPairs(const Sample & inputSample, const Sample & outputSample)
```

Comment:
The static method should be renamed, because it does not return the same output as the current `DrawPairs` static method.
```suggestion
GridLayout VisualTest::DrawPairsXY(const Sample & inputSample, const Sample & outputSample)
```